src/universal/modules/meeting/components/HelpTextForTeam.js

```diff
@@ -10,7 +10,7 @@ const HelpTextForTeam = (props) => {
   const handleAgendaControl = () => {
     agendaInputRef.focus();
   };
-  const isCheckedInFalse = Boolean(currentTeamMember.isCheckedIn === false);
+  const isCheckedInFalse = currentTeamMember.isCheckedIn === false;
```

Comment:
@mattkrick I was looking for a `false` instead of a `null` so that `isCheckedInFalse` only works with the user marks a teammate as “Not Here” — will this still work like that? Was wrestling with `null` which is what you get when you don’t mark anything. Didn’t want to make any assumptions if the user didn’t mark anything.

Reply:
Looks like this works as expected! I don’t totally understand `Boolean()` but I assume it is redundant in this case.

Reply:
it's just redundant. if i ask you does 1 equal 2?, you're already going to tell me `false`. No need to coerce false to a boolean if it already is one.

If i say something like `const arrayHasa4 = [1,2,3].find(x => x === 4)` then it'll return undefined because nothing was found. If we want that variable to be a boolean, we'd wrap it in a Boolean because `Boolean(undefined) === false`.